vizro-core/tests/unit/vizro/models/_components/form/test_slider.py

```diff
@@ -3,24 +3,47 @@
 
 import plotly
 import pytest
-from dash import dcc
+from dash import dcc, html
 from pydantic import ValidationError
 
 import vizro.models as vm
 
 
 @pytest.fixture()
 def expected_slider():
-    return dcc.Slider(
-        min=0,
-        max=10,
-        step=1,
-        marks={},
-        value=5,
-        included=False,
-        className="slider_control",
-        id="slider_id",
-        persistence=True,
+    return html.Div(
```

Comment:
We should always test the entire component and not only the dcc.component e.g. the dcc.Inputs are also quite important. If someone were to change something there (e.g. renaming the ID), the slider callbacks might not work properly if not adjusted.